src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+#include "pal_types.h"
+
+#ifdef  GSSFW
```

Comment:
Rather than include these in the header file (which then makes it easy to have circular header includes once the project size increases), please move this include into the CPP file and forward declare the structs you need (if possible)